lib/screens/home/widgets/drawer/drawer_header.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+
+class MenuDrawerHeader extends StatelessWidget {
+
+  final String name;
+  final String email;
+  final CircleAvatar photo;
+
+  //Default values
+  static const defaultPhotoText = 'defaultText';
```

Comment:
I think (but i'm not 100% sure) that having static variables like these inside a class in Dart will implicitly generate getters (and technically setters, but they'll just produce a compile-time error). You can make variables private to their scope in Dart by adding an underscore in the beginning, i.e. the above would be _defaultPhotoText.

I might be wrong about this but it was just something that came to mind as I was skimming the code. No need to change it for this PR though ofc.

Reply:
Well, I don't think having static variables like these inside a class in Dart will implicitly generate getter... Here is what I saw in the documentation "Use const for variables that you want to be compile-time constants. If the const variable is at the class level, mark it static const. Where you declare the variable, set the value to a compile-time constant such as a number or string literal, a const variable, or the result of an arithmetic operation on constant numbers"  

And according to Android Studio  named optional variable can not start with an underscore. I didn't worry so much about all this because, there has not been any concrete decision about the login data or even "longing in".

Reply:
`static const defaultPhotoText = 'defaultText';` generates a getter as a class variable. It's accessed  by calling `MenuDrawerHeader.defaultPhotoText`. Changing the variable inside MenuDrawerHeader class into  `static const _defaultPhotoText = 'defaultText';` and the constructor into `MenuDrawerHeader({this.name = _defaultPhotoText, this.email= _defaultPhotoText,
    this.photo = defaultPhoto});` will make the variable inaccessible outside the MenuDrawerHeader widget itself.

I'm just pointing this out as an example of how dart and flutter works. It's good form to use private variables whenever possible. if a variable name starts with an underscore it means the variable is private to that particular scope.

Reply:
I get your point(encapsulation right?) and I tried that from the beginning, but the IDE kept given an error.  You can confirm by trying it yourself.   And like I wrote earlier, I am not worried about that right now because all is not decided.  

I must point out that I am not a fan of magic values and that is why I have static const defaultPhotoText = 'defaultText'; as opposed to just ({this.name = 'defaultText'})

Reply:
Ok, I get your point, I will make the variables private. Anyway, I wasn't worried about that cos all is not yet decided.